OpenRA.Mods.Common/Widgets/Logic/MissionBrowserLogic.cs

```diff
@@ -286,14 +289,16 @@ void SelectMap(MapPreview preview)
 
 			if (difficultyButton != null)
 			{
-				var difficultyName = new CachedTransform<string, string>(id => id == null || !difficulties.ContainsKey(id) ? "Normal" : difficulties[id]);
+				var difficultyName = new CachedTransform<string, string>(id => id == null || !difficulties.ContainsKey(id) ?
+					preview.GetLocalisedString(DifficultyNormal) : preview.GetLocalisedString(difficulties[id]));
```

Comment:
```suggestion
					preview.GetLocalisedString(DifficultyNormal : difficulties[id]));
```

Might be worth reducing duplication (Ofc also with inlining the condition)